### PR TITLE
reinforced walls can't be burnt by clf3 anymore

### PR DESCRIPTION
**Pull request by ShadowDeath6**
if adam complains about me nerfing this I'm gonna be pretty mad

:cl: ShadowDeath6
rscdel: Reinforced walls can't be burnt by chlorine triflouride anymore.
/:cl:

### CODE REVIEW

**Comment by McDonald072** [comment]
he did it the madman

File: code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm [hunk 1]
```diff
@@ -69,6 +69,8 @@
 					PoolOrNew(/obj/effect/hotspot, F)
 	if(istype(T, /turf/closed/wall/))
 		var/turf/closed/wall/W = T
+		if(W.walltype = "rwall") //use thermite
```
Comment (Super3222) [inline code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm:72]: good change.

**Comment by ShadowDeath6** [comment]
I see that :-1: @McDonald072 why do you feel this way

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
I can't powergame my way into AI core in 10 seconds anymore

**Comment by Jc1469** [comment]
hi it me.  balance plis.  
balance plis approve.  thermite should be used because it doesn't take a nanosecond and has counterplay. hi it me.

**Pull request merged**